apps/www/components/Questionnaire/Submissions/Submission.js

```diff
@@ -115,6 +116,7 @@ export const SubmissionAuthor = ({
       {displayAuthor.profilePicture && (
         <img
           {...styles.headerPicture}
+          style={{ ...customStylePicture }}
```

Comment:
```suggestion
          style={customStylePicture}
```

shouldn't this work too?